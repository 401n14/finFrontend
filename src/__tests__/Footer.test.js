import React from 'react';
import Footer from '../components/Footer';
import {configure, mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

configure({adapter:new Adapter()});

describe('Snapshot Test', () => {
    it('renders correctly, Snapshot', () => {
        const page = renderer.create(<Footer />).toJSON();
        expect(page).toMatchSnapshot();
    });
});