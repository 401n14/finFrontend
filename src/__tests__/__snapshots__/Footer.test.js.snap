// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test renders correctly, Snapshot 1`] = `
<footer
  className="bg-light p-3 text-center"
>
  <p
    style={
      Object {
        "marginTop": "2.5em",
      }
    }
  >
    <b>
      nightJAMMM
    </b>
     team Â© CodeFellows | 2020
  </p>
</footer>
`;
